e2e/protractor.excludes.json

```diff
@@ -15,5 +15,6 @@
  "C586780" : "Include once ACA starts using ACS 7+, https://issues.alfresco.com/jira/browse/ACA-3601",
  "C586781" : "Include once ACA starts using ACS 7+, https://issues.alfresco.com/jira/browse/ACA-3601",
  "C280132" : "Shared files list is not empty on the live environment",
- "C280094" : "Shared files list is not empty on the live environment"
+ "C280094" : "Shared files list is not empty on the live environment",
+ "MNT-21058" : "https://alfresco.atlassian.net/browse/ACA-4145"
```

Comment:
this MNT is closed. 
Is it reproducing again?

Reply:
it's a bit weird to have 2 bug IDs here...

Reply:
it's ok one is excluded is correct the other is the name that was given wrong . We should have a testrail instead to point to Jira